src/tables/math.rs

```diff
@@ -0,0 +1,1062 @@
+const FRACTION_DENOMINATOR_SHIFT_DOWN_OFFSET: usize = 128;
+const FRACTION_DENOMINATOR_DISPLAY_STYLE_SHIFT_DOWN_OFFSET: usize = 132;
+const FRACTION_NUMERATOR_GAP_MIN_OFFSET: usize = 136;
+const FRACTION_NUM_DISPLAY_STYLE_GAP_MIN_OFFSET: usize = 140;
+const FRACTION_RULE_THICKNESS_OFFSET: usize = 144;
+const FRACTION_DENOMINATOR_GAP_MIN_OFFSET: usize = 148;
+const FRACTION_DENOM_DISPLAY_STYLE_GAP_MIN_OFFSET: usize = 152;
+const SKEWED_FRACTION_HORIZONTAL_GAP_OFFSET: usize = 156;
+const SKEWED_FRACTION_VERTICAL_GAP_OFFSET: usize = 160;
+const OVERBAR_VERTICAL_GAP_OFFSET: usize = 164;
+const OVERBAR_RULE_THICKNESS_OFFSET: usize = 168;
+const OVERBAR_EXTRA_ASCENDER_OFFSET: usize = 172;
+const UNDERBAR_VERTICAL_GAP_OFFSET: usize = 176;
+const UNDERBAR_RULE_THICKNESS_OFFSET: usize = 180;
+const UNDERBAR_EXTRA_DESCENDER_OFFSET: usize = 184;
+const RADICAL_VERTICAL_GAP_OFFSET: usize = 188;
+const RADICAL_DISPLAY_STYLE_VERTICAL_GAP_OFFSET: usize = 192;
+const RADICAL_RULE_THICKNESS_OFFSET: usize = 196;
+const RADICAL_EXTRA_ASCENDER_OFFSET: usize = 200;
+const RADICAL_KERN_BEFORE_DEGREE_OFFSET: usize = 204;
+const RADICAL_KERN_AFTER_DEGREE_OFFSET: usize = 208;
+const RADICAL_DEGREE_BOTTOM_RAISE_PERCENT_OFFSET: usize = 212;
+
+impl<'a> MathConstants<'a> {
+    /// Percentage of scaling down for level 1 superscripts and subscripts. Suggested value: 80%.
+    #[inline]
+    pub fn script_percent_scale_down(&self) -> i16 {
+        Stream::read_at(self.data, SCRIPT_PERCENT_SCALE_DOWN_OFFSET).unwrap_or(0)
+    }
+
+    /// Percentage of scaling down for level 2 (scriptScript) superscripts and subscripts.
+    /// Suggested value: 60%.
+    #[inline]
+    pub fn script_script_percent_scale_down(&self) -> i16 {
+        Stream::read_at(self.data, SCRIPT_SCRIPT_PERCENT_SCALE_DOWN_OFFSET).unwrap_or(0)
+    }
+
+    /// Minimum height required for a delimited expression (contained within parentheses, etc.) to
+    /// be treated as a sub-formula. Suggested value: normal line height × 1.5.
+    #[inline]
+    pub fn delimited_sub_formula_min_height(&self) -> u16 {
+        Stream::read_at(self.data, DELIMITED_SUB_FORMULA_MIN_HEIGHT_OFFSET).unwrap_or(0)
+    }
+
+    /// Minimum height of n-ary operators (such as integral and summation) for formulas in display
+    /// mode (that is, appearing as standalone page elements, not embedded inline within text).
+    #[inline]
+    pub fn display_operator_min_height(&self) -> u16 {
+        Stream::read_at(self.data, DISPLAY_OPERATOR_MIN_HEIGHT_OFFSET).unwrap_or(0)
+    }
+
+    /// White space to be left between math formulas to ensure proper line spacing. For example,
+    /// for applications that treat line gap as a part of line ascender, formulas with ink going
+    /// above (os2.sTypoAscender + os2.sTypoLineGap - MathLeading) or with ink going below
+    /// os2.sTypoDescender will result in increasing line height.
+    #[inline]
+    pub fn math_leading(&self) -> MathValueRecord {
```

Comment:
You should return `MathValueRecord<'a>` to extend the lifetime.

Reply:
Yes, indeed I should. These were automatically generated with a script, and I forgot to review these lines after the successful compilation. Changed now.